tfexec/terraform_workspace_show_test.go

```diff
@@ -0,0 +1,72 @@
+package tfexec
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestTerraformCLIWorkspaceShow(t *testing.T) {
+	cases := []struct {
+		desc         string
+		mockCommands []*mockCommand
+		want         string
+		ok           bool
+	}{
+		{
+			desc: "parse output of terraform workspace show",
+			mockCommands: []*mockCommand{
+				{
+					args:     []string{"terraform", "workspace", "show"},
+					stdout:   "default",
+					exitCode: 0,
+				},
+			},
+			want: "default",
+			ok:   true,
+		},
+		{
+			desc: "with existing workspace",
```

Comment:
desc: "with an error",

Reply:
not sure what you mean here because the second test case doesn't expect an error from the terraform command. I've added a third test case to test for an error returned by `terraform workspace select`

Reply:
My mistake. I guess when I wrote this comment, I probably misread the 2nd test case returns an error, but now I'm reading the 2nd test case with fresh eyes, apparently I was wrong. Sorry. Although, it's good to add the 3rd error case you suggested. Thanks!